lib/submitorder.rb

```diff
@@ -0,0 +1,32 @@
+require './lib/sendtext'
+
+class SubmitOrder
+
+  attr_reader :client, :total
+  attr_accessor :order
+
+  def initialize sendtext_klass=Sendtext, account_sid, auth_token
```

Comment:
Style/OptionalArguments: Optional arguments should appear at the end of the argument list. (https://github.com/bbatsov/ruby-style-guide#optional-arguments)